frontend/src/reducers/index.js

```diff
@@ -1,6 +1,8 @@
 import { combineReducers } from 'redux';
 import emptyReducer from './empty';
+import user from './user';
```

Comment:
Can the reducer be named something like "authenticationReducer" or "authReducer"